cli/main.rs

```diff
@@ -49,8 +49,7 @@ use crate::worker::Worker;
 use deno::v8_set_flags;
 use flags::DenoFlags;
 use flags::DenoSubcommand;
-use futures::lazy;
-use futures::Future;
+use futures::{future, lazy, Future};
```

Comment:
our coding style is not use do this and to keep each use per line, as it makes it easier to add an remove

Reply:
Thanks. I'll fix this